Updates to Operator installation procedure (#221)
redhat-developer/app-services-operator: docs/installation.adoc

== Installing the RHOAS Operator on OpenShift

This procedure shows how to use OperatorHub to install the OpenShift Application Services (RHOAS) Operator.

NOTE: To install the RHOAS Operator, you  must have cluster administrator privileges for your OpenShift cluster.

.Procedure
. Log in to the OpenShift web console as a cluster administrator.
. Use the perspective switcher in the top-left corner to switch to the *Administrator* perspective.
. In the left menu, click *Operators* > *OperatorHub*.
. In the *Filter by keyword...* field, enter `RHOAS`.
. Select the `OpenShift Application Services (RHOAS)` Operator.
. If a dialog box entitled `Show community Operator` appears, review the included information. When you have finished reviewing the information, click *Continue*.
+

> An information pane for the RHOAS Operator opens.

Reviewer: pane => `sidebar`

. Review the information about the RHOAS Operator and click *Install*.

. On the *Install Operator* page:
.. For the *Installation mode* option, ensure that `All namespaces on the cluster` is selected.
.. For the `Update channel` and `Update approval` options, keep the default values.
.. Click *Install*.
. When the installation process finishes, click *View Operator* to see the Operator details. Alternatively, click *Operators* > *Installed Operators* and then click the RHOAS Operator.
+
Observe that the RHOAS Operator is installed in the `openshift-operators` namespace.

Once the Operator is installed, you can use the `rhoas` CLI to interact with the Operator using `rhoas cluster` commands.

In addition, you can use the Developer Catalog in the OpenShift web console to create resource instances for the Operator. To access the Developer Catalog, use the perspective switcher in the top-left corner to switch to the *Developer* perspective. Then, click *+Add* > *From Catalog*.

=== Installing preview versions of the Operator

> You can install preview versions of the Operator using the CatalogSource for those versions. If you are a developer that wants to try an earlier version of the Operator, see the link:./build_docs/dev-install.adoc[contributor installation documentation].

Reviewer: ```suggestion
You can install early preview versions of the Operator using the custom CatalogSource. If you are a developer that wants to try an earlier version of the Operator, see the link:./build_docs/dev-install.adoc[contributor installation documentation].
```
